### ISSUE TITLE
Bundler: You must use Bundler 2 or greater with this lockfile

### ISSUE THREAD

**Issue by raghubetina** [firstdraft/appdev_template#100]
@jelaniwoods I created a brand new project with appdev_template and then created a Gitpod workspace based upon it, then received an error:

```
Bundler: You must use Bundler 2 or greater with this lockfile
```

which terminated `bin/setup`. I resolved it quickly by adding

```ruby
  system! "gem install bundler"
  system! "gem update bundler"
```

before

```ruby
  system! "bundle install"
```

in `bin/setup`.

Task: reproduce the error and decide what the best resolution is.

**Comment by jelaniwoods** [comment]
This is interesting, since `omnicalc-api` was generated with the template last week and caused no issues.

**Comment by jelaniwoods** [comment]
I'm not sure if this is exactly a template issue. This happens when the person who generates the new app uses Bundler version 2. Apparently the docker image doesn't have that version so that could be updated. This feels kind of like #93, where we should decide what version of Bundler to use for all projects so this doesn't happen.

However, bundler can be really annoying and I don't want to have to specify which version I'm using for other projects and which version I need to use for appdev assignments. What are your thoughts @raghubetina ?

**Comment by raghubetina** [comment]
@jelaniwoods I suppose the ultimate way to do it would be to include and keep updated a Dockerfile with every project, and then upload an image to Dockerhub for every project.

Alternatively we could update the single Docker image that we're using to include some latest version of Bundler.

Or the solution I posted, to install/update Bundler before `bundle install`, ought to work?

**Comment by jelaniwoods** [comment]
This should be fixed in #109 .
